docs/building-a-form/common-definitions.md

```diff
@@ -45,22 +47,25 @@ Available definitions are:
 ### Autosuggest
 
 A common type-ahead widget that lets a user type in values and narrow down a longer list of options. It is most commonly used with an `enum` of the available options as shown here. Define the uiSchema by calling the function that you import. You can pass an object with additional uiSchema properties.
+
 ```js
 import { uiSchema as autosuggestUI } from 'us-forms-system/lib/js/definitions/autosuggest';
```

Comment:
This import statement needs to change to `import { autosuggestConfig } from 'us-forms-system/lib/js/definitions/autosuggest';`.